src/components/ViewServiceDetails/ViewServiceDetails.js

```diff
@@ -22,43 +22,34 @@ class ViewServiceDetails extends React.Component {
 		};
 	}
 
-	componentDidMount(){
+	componentDidMount() {
 		this.getVehiclesList();
 		let queryParams = this.props.location.query;
-		if( queryParams && queryParams.vehicle ){
-			this.chooseVehicle(null,null, queryParams.vehicle);
+		if (queryParams && queryParams.vehicle) {
+			this.chooseVehicle(queryParams.vehicle);
 		}
 	}
 
 	render() {
+		let placeHolderItem = <MenuItem disabled value="" key='chooseVehicle'><em>Choose Vehicle</em></MenuItem>;
+		const menuItems = [placeHolderItem, ...this.state.vehicles.map((vehicle) => (
+			<MenuItem value={vehicle._id} key={vehicle._id} >
+				{vehicle.name}
+			</MenuItem>
+		))];
+
 		return (
 			<div className={styles['service-details']}>
-				<Header title={'View service details'}/>
+				<Header title={'View service details'} />
 				<div className={styles['body']}>
 					<div className={globalStyles['row']}>
-						<Select 
-							native 
-							// displayEmpty 
-							value={this.state.currentVehicle} 
-							onChange={this.chooseVehicle.bind(this)}
-							renderValue={selected => {
-								// debugger;
-								if (selected && selected.length === 0) {
-									return <em>Placeholder</em>;
-								}
-								return selected.join(', ');
-							}}
+						<Select
+							displayEmpty
```

Comment:
Understand what displayEmpty is for and then use accordingly.

Reply:
it allows displaying default value (currently it is displaying 'choose vehicle'), if nothing is selected